src/main/java/eu/deltasource/internship/abankingsystem/repository/bankInstitutionRepository/BankInstitutionRepository.java

```diff
@@ -0,0 +1,40 @@
+package eu.deltasource.internship.abankingsystem.repository.bankInstitutionRepository;
+
+import eu.deltasource.internship.abankingsystem.model.BankAccount;
+import eu.deltasource.internship.abankingsystem.model.BankInstitution;
+
+public interface BankInstitutionRepository {
+
+    /**
+     * Save bank to Map.
+     *
+     * @param bankInstitution bank object to be added to Map
+     */
+    void addBankToMap(final BankInstitution bankInstitution);
+
+    /**
+     * Get bank from Map by id
```

Comment:
I think you should not mention the implementation details like "Map" in this use case